packages/concerto-core/lib/decoratormanager.js

```diff
@@ -432,6 +433,14 @@ class DecoratorManager {
                     : (decorated.decorators = [newDecorator]);
             }
         } else if (type === 'APPEND') {
+            decorated.decorators.forEach((d)=>{
+                if(d === newDecorator){
```

Comment:
We should match against the decorator name instead. Two decorators with the same name but different arguments should still be invalid.

Reply:
hey @mttrbrts done this and I am still confused about the test I have written... can you help out?